PWGJE/Core/JetDerivedDataUtilities.h

```diff
@@ -383,6 +383,15 @@ float trackEnergy(T const& track, float mass = mPion)
   return std::sqrt((track.p() * track.p()) + (mass * mass));
 }
 
+template <typename T>
+bool selectTrackDcaZ(T const& track, double dcaZmax = -99.)
+{
+  if (dcaZmax < 0) {
```

Comment:
do we need this condition?
also did you check if dca can be negative?

Reply:
we can just put the default value to 99 or something then it will always be true?